docs/contribute/file_format.md

```diff
@@ -17,17 +17,11 @@ for and how to use it. This is what this one looks like:
 
 # Version: (system version)
 # Created: jj/mm/yyyy (minecraft version)
-# Last verification: jj/mm/yyyy (minecraft version)
 # Last modification: jj/mm/yyyy (minecraft version)
 
-# Original path : bs.path:to/function
 # Documentation : https://link.to/documentation/page
```

Comment:
The parentheses are confusing.
In the case of `Version`, the expected value is `1.0`. For example: `Version: 1.0`.
In the case of `Last modification`, the expected value is `(1.20)`. For example: `05/08/2023 (1.20)`.
Maybe we have to illustrate the format through a kind of "templating" syntax?

Example:
```
# INFO ------------------------------------------------------------------------
# Copyright © <current year> Gunivers Community.

# Authors       : <<author 1>, <author 2>, ...>
# Contributors  :  <<contributor 1>, <contributor 2>, ...>

# Version: <system version>
# Created: <jj>/<mm>/<yyyy> (<minecraft version>)
# Last modification: <jj>/<mm>/<yyyy> (<minecraft version>)

# Documentation : <https://link.to/documentation/page>
# Note          : <some notes>

# CODE ------------------------------------------------------------------------
```
wdyt?